renku_notebooks/api/auth.py

```diff
@@ -0,0 +1,107 @@
+import json
+import logging
+import os
+import warnings
+from functools import wraps
+
+import requests
+from flask import Blueprint, current_app, jsonify, request, make_response, redirect, abort
+
+from jupyterhub.services.auth import HubOAuth
+
+from .. import config
+from ..util._kubernetes import annotate_servers
+
+try:
+    auth = HubOAuth(
+        api_token=os.environ.get('JUPYTERHUB_API_TOKEN'),
+        cache_max_age=60,
+    )
+except ValueError as e:
```

Comment:
You are importing `auth` in other module. If an exception is raised here, then `auth` is not defined and we will get an error when trying to import it. I would rather fail explicitly here or put `auth = None` in the `except` block.

Reply:
thanks, I had that there during the refactor and forgot to remove it.